pkg/server/endpoints/node/handler.go

```diff
@@ -53,17 +55,31 @@ type Handler struct {
 	c       HandlerConfig
 	limiter Limiter
 
-	dsCache *datastoreCache
+	dsCache                          *datastoreCache
+	fetchRegistrationEntriesCache    *regentryutil.FetchRegistrationEntriesCache
+	useFetchRegistrationEntriesCache bool
 }
 
 func NewHandler(config HandlerConfig) *Handler {
 	if config.Clock == nil {
 		config.Clock = clock.New()
 	}
+	fetchX509SVIDCache, err := regentryutil.NewFetchX509SVIDCache(fetchSVIDCacheSize)
+	if err != nil {
+		config.Log.Errorf("could not create lru cache. Not using cache for FetchX509SVID", err)
```

Comment:
Please use field-based logging, see note in [CONTRIBUTING.md](https://github.com/spiffe/spire/blob/master/CONTRIBUTING.md#logs-and-errors).

e.g.
```
config.Log.WithError(err).Error(msg)
```

Reply:
Log messages should also start with a capital letter.